sql/analyzer/indexes.go

```diff
@@ -261,6 +262,59 @@ func getIndexes(
 		return result, nil
 	}
 
+	// TODO: maybe this block can just go into previous switch
+	// TODO (james): add all other spatial index supported functions here
+	// TODO: make generalizable to all functions?
+	switch e := e.(type) {
+	case *spatial.Intersects:
```

Comment:
missing `getNegatedIndexes` and `getMultiColumnIndexes`, or if not supporting at least test those to make sure they don't break